mobify/progressive-web-scaffold
# Refactor Sign-in & Register Skeletons

Refactor the Sign-in and Register Skeletons

## Changes
- Improve skeletons for Sign-In and Register pages
- Improved button background transitions (i.e. from disabled to enabled)

| Before | After |
| --- | --- |
|  ![signin-before](https://cloud.githubusercontent.com/assets/734535/20692526/730e23b2-b58d-11e6-81bd-0c8c4b59cfaf.gif) | ![signin-after](https://cloud.githubusercontent.com/assets/734535/20692534/80fa1684-b58d-11e6-8b0e-12b3ecbabdae.gif) |

| Before | After |
| --- | --- |
| ![register-before](https://cloud.githubusercontent.com/assets/734535/20692531/7958b1e2-b58d-11e6-9cc7-2aaa054b999d.gif) | ![register-after](https://cloud.githubusercontent.com/assets/734535/20692542/878dcdce-b58d-11e6-8002-e2b6f436bb1c.gif) |

## How to test-drive this PR
- Run `npm run dev`
- [Preview Merlins Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2Feye-of-newt.html&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Visit the [Sign-in page](https://www.merlinspotions.com/customer/account/login/)
- Visit the [Register page](https://www.merlinspotions.com/customer/account/create/)
- Observe the page skeletons as they load and ensure all of them appear consistent in proportion to their final content.
